dev.html

```diff
@@ -2377,6 +2476,60 @@ <h2>Referring to other mappings</h2>
         </div>
       </section>
 
+       <p> The joins can also be declared using inline <a href="#functions">functions</a> in a more compact way. If more than one condition is required, the <code>equal</code> function can be repeated several times. The <code>join</code> and <code>equal</code> do not need prefix as they are specific functions of R2RML/RML. If no parameters name are provide the order MUST be <code>parent mapping reference</code>, and within equal function <code>str1 (or s)</code> first and <code>str2 (or o)</code> second. </p>
```

Comment:
Can we move this extension to a separate PR? Of course these things are all about preferences, but I don't really so how the proposal is that much clearer than

```yaml
mappings:
  person:
    subjects: http://example.com/person/$(ID)
    predicateobjects:
      - predicates: foaf:worksFor
        objects:
          - mapping: project
            function: equal(str1=$(projectID),str2=$(ID)))
```

Amount of characters (even if that's horrible measure) is exactly the same, but you don't create new constructs.
Also 'equal' is not a specific function of [R2]RML, the equal function is differently translated to RML depending on where's it's used in the YARRRML.

If you introduce the `mapping=[mappingName]` struct, you kind of overload the already existing `[parameter]=[parameterValue]` struct used for the function description.

Reply:
I would create a new specific branch with this proposed upgrade for making a new PR so we can discuss it in deep. But I don't see how multiple conditions for one join can fit in this approach. Indeed, `mapping` is treated as a parameter in the proposal

Reply:
I realized that if I make a new branch, it will have conflicts with this one (if I don't include the `star` part) as they are the same file. So: i) we can wait to decide on the `star` part and then start the discussion with `join`; ii) continue the discussion here

Reply:
this part has removed until we merge this issue